github/git/pull.js

```diff
@@ -1,37 +1,53 @@
 const path = require('path');
 const simpleGit = require('simple-git');
 
-const ORIGIN = 'origin';
-const MASTER = 'master';
-const MAIN = 'main';
-
 const git = simpleGit();
 
+const DEFAULT_ORIGIN = 'origin';
+// By default, GitHub names the default branch main in any new repository.
+const DEFAULT_BRANCH = 'main';
+
+const findDefaultOriginAndBranch = async () => {
+  // Answers in the form of `origin/branch`
+  const gitSymbolicCmd = await git.raw(
+    'symbolic-ref',
+    '--short',
+    'refs/remotes/origin/HEAD',
+  );
+  const originAndBranch = gitSymbolicCmd.trim().split('/');
+
+  return {
+    origin: originAndBranch[0] ?? DEFAULT_ORIGIN,
+    branch: originAndBranch[1] ?? DEFAULT_BRANCH,
+  };
```

Comment:
Small syntax suggestion :)

```suggestion
  const [origin = DEFAULT_ORIGIN, branch = DEFAULT_BRANCH] = gitSymbolicCmd.trim().split('/');

  return {
    origin,
    branch,
  };
```

Reply:
Oh! Good idea, it is way cleaner that way 👌